src/hacbs/components/ApplicationDetails/tabs/overview/visualization/utils/draw-utils.ts

```diff
@@ -2,7 +2,7 @@ import { Point } from '@patternfly/react-topology';
 import { DrawDesign, NODE_SEPARATION_HORIZONTAL } from '../const';
 
 type SingleDraw = (p: Point) => string;
-type DoubleDraw = (p1: Point, p2: Point, indentedStart?: number) => string;
+type DoubleDraw = (p1: Point, p2: Point, indentedStart?: number, indentedEnd?: number) => string;
```

Comment:
We should create an issue for PF topology to handle this and remove this file completely when it does.